client/start.js

```diff
@@ -348,11 +348,9 @@ export function startApp(renderApplication, reducers, opts) {
     registerPageView(store.getState().qt);
 
     if (page.title) {
-      global.document.title = get(
-        page,
-        ["data", "story", "seo", "meta-title"],
-        page.title
-      );
+      global.document.title =
+        get(page, ["data", "customSeo", "title"], undefined) ||
```

Comment:
do you need the undefined, the fallback can be `get(page, ["data", "story", "seo", "meta-title"], page.title);` right ?

Reply:
yes, it could be but I thought I will be 3 levels get so I added undefined followed by fallback. if you want i can move this inside as fallback.